newrelic/docs-website
# feat: create `InlinePopover` component

also sets up the `TwoButton` layout (currently the only one), and makes it easy to add more

## testing
- pick your favorite MDX file
- add `<InlinePopover type="apm" />`, or one of the other `type`s in `popovers.json`
- check that page and make sure the popover renders correctly and Tessen events are sent
